src/vmm/Cargo.toml

```diff
@@ -30,11 +30,13 @@ versionize_derive = "0.1.5"
 vm-allocator = "0.1.0"
 vm-fdt = "0.2.0"
 vm-superio = "0.7.0"
-log = { version = "0.4.17", features = ["serde"] }
+log = { version = "0.4.17", features = ["std", "serde"] }
+
+aes-gcm =  { version = "0.10.1", default-features = false, features = ["aes"] }
+base64 = "0.13.0"
+bincode = "1.2.1"
+micro_http = { git = "https://github.com/firecracker-microvm/micro-http" }
```

Comment:
Why is there a newline separating these?

Reply:
initially I thought to keep the newly added dependencies together so they were separated by a newline.

Reply:
removed the newline.